packages/autorest.typescript/package.json

```diff
@@ -75,7 +75,7 @@
     "source-map-support": "^0.5.16",
     "ts-morph": "^15.1.0",
     "@azure/core-auth": "^1.6.0",
-    "@azure-tools/rlc-common": "workspace:^0.22.0"
+    "@azure-tools/rlc-common": "workspace:^0.23.0"
```

Comment:
We should probably upgrade the version in `packages/autorest.typescript/package.json` too. and add the change log here to keep consistent.

Reply:
you mean bump the version of autorest.typescript?

Reply:
Pushed a change for this